src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs

```diff
@@ -162,6 +172,36 @@ public void MetricObserverCallbackError(string metricName, string exception)
             this.WriteEvent(16, metricName, exception);
         }
 
+        [Event(17, Message = "Batcher finished collection with '{0}' metrics.", Level = EventLevel.Informational)]
+        public void BatcherCollectionCompleted(int count)
+        {
+            this.WriteEvent(17, count);
+        }
+
+        [Event(18, Message = "Collection completed in '{0}' msecs.", Level = EventLevel.Informational)]
+        public void CollectionCompleted(long msec)
+        {
+            this.WriteEvent(18, msec);
+        }
+
+        [Event(19, Message = "Exception occured in Metric Controller. Exception: '{0}'", Level = EventLevel.Warning)]
```

Comment:
Can you please include the result of the problem into the message. Like "Exception occured in Metric Controller. Metric Controller will not proceed 1 batch of measurements. Exception: " or something like this.